### DISCUSSION TITLE
Allow Multi-series Stat/Gauge panels to be laid out as if they were repeated single-series panels

### DISCUSSION THREAD

**Discussion by philomory** [grafana/grafana#38233] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
I'd like to be able to define a single query which returns multiple series, and have those series rendered similarly to a repeated Stat or Gauge panel where each panel displays a single series. In other words, I'd like to be able to define a query like this:

```flux
from(bucket: "example")
  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
  |> filter(fn: (r) => r._measurement == "cpu" and r._field == "usage_idle" and r.cpu == "cpu-total")
  |> group(columns: ["host"])
  |> mean()
```
and get a view like this:

![image](https://user-images.githubusercontent.com/200123/109706143-aae01b00-7b3c-11eb-9650-96fb9788f362.png)

instead of this:
![image](https://user-images.githubusercontent.com/200123/109705476-ea5a3780-7b3b-11eb-8b9a-30b52e597456.png)

**Why is this needed**:
With repeated panels, the more series you are displaying, the more space the panels take up; but each panel is a fixed size and so they are always readable without wasting any space. With a single Stat or Gauge panel showing multiple series, the panel is a fixed size, and so the more series there are to display, the smaller each individual guage/stat display gets, eventually becoming unreadable.

On the other hand, when using the existing "Repeated Panel" feature that is based on the use of template variables, in order to display 250 hosts Grafana will issue 250 separate queries; it is _much_ more efficient to just issue a single query that returns multiple series, instead.

Note that it's not actually critical that each series get it's own _proper_ panel with its own title bar, drop-down menu, etc; it would be sufficient if the Gauge and Stat panels offered options to fix the size of individual display elements and then the panel would grow and shrink automatically depending on the number of series returned by the query, if that's easier to implement.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
